pandas/core/indexes/base.py

```diff
@@ -1955,6 +1955,12 @@ def is_mixed(self) -> bool:
         >>> idx.is_mixed()
         False
         """
+        warnings.warn(
+            "Index.is_mixed is deprecated and will be removed in a future version. "
+            "Check index.inferred_type directly instead.",
+            FutureWarning,
+            stacklevel=2,
```

Comment:
Is the stack level required here? Wouldn’t it be thrown when calling directly without it?

Reply:
A stacklevel is always required (and when the user would call it directly, it needs to be 2). The default in the stdlib is kind of useless